qemu/tests/cfg/chardev_pci_serial_login.cfg

```diff
@@ -0,0 +1,6 @@
+- chardev_pci_serial_login:
+    only RHEL
+    only x86_64
+    type = chardev_pci_serial_login
+    serial_type = pci-serial
```

Comment:
we can't use pci-serial and isa-serial to login guest in the same time, there is only one console can work and the other will be blocked.